run_example.py

```diff
@@ -1,38 +1,53 @@
 """Example of running SpinUp Bis algorithms."""
-
+import argparse
 import os
 
 import gym
 import tensorflow as tf
 
 from spinup_bis import sac_tf2 as agent  # pylint: disable=import-only-modules
+from spinup_bis import sop_tf2 as agent_sop  # pylint: disable=import-only-modules
```

Comment:
Do not change the whole script. Because you can simply swap `sac_tf2` for `sop_tf2` and it should work, just leave the suggested line commented and revert all of the other changes. If someones want to, they will be able to uncomment it and run SOP instead of SAC.
```suggestion
# from spinup_bis import sop_tf2 as agent  # pylint: disable=import-only-modules
```

Reply:
done